dev/lib/product_taxonomy/commands/generate_release_command.rb

```diff
@@ -107,23 +114,43 @@ def run_git_command(*args)
     end
 
     def git_repo_root
-      @git_repo_root ||= `git rev-parse --show-toplevel`.strip
+      @git_repo_root ||= %x(git rev-parse --show-toplevel).strip
     end
 
     def get_commit_hash(ref)
-      `git rev-parse --short #{ref}`.strip
+      %x(git rev-parse --short #{ref}).strip
     end
 
-    def update_integration_mappings
+    def update_integration_mappings_stable_release
+      logger.info("Updating mapping files for stable release version: #{@version}")
+      final_shopify_taxonomy_string = "shopify/#{@version}"
+
+      logger.info("Updating to_shopify.yml files -> output_taxonomy: #{final_shopify_taxonomy_string}")
       Dir.glob(File.expand_path("integrations/**/mappings/to_shopify.yml", ProductTaxonomy.data_path)).each do |file|
+        logger.info("Processing file: #{file}")
         content = File.read(file)
-        content.gsub!(/output_version: .*?-unstable/, "output_version: #{@version}")
+        content.gsub!(%r{output_taxonomy: shopify/\d{4}-\d{2}(-unstable)?}, "output_taxonomy: #{final_shopify_taxonomy_string}")
         File.write(file, content)
       end
 
+      logger.info("Updating from_shopify.yml files -> input_taxonomy: #{final_shopify_taxonomy_string}")
+      Dir.glob(File.expand_path("integrations/**/mappings/from_shopify.yml", ProductTaxonomy.data_path)).each do |file|
+        logger.info("Processing file for input_taxonomy update: #{file}")
+        content = File.read(file)
+        content.gsub!(%r{input_taxonomy: shopify/\d{4}-\d{2}(-unstable)?}, "input_taxonomy: #{final_shopify_taxonomy_string}")
+        File.write(file, content)
+      end
```

Comment:
There's an exact copy of this code below in `update_integration_mappings_next_unstable`, can we extract it into a shared method?